src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -188,435 +188,730 @@ If a problem occurs, check output in /var/log/proactive/scheduler
 
 WARNING: Once the upgrade is performed and the new version started for the first time, the ProActive server will upgrade <<_glossary_catalog,Catalog>> objects to the new version. This procedure ensures that any object modified manually by a user will not be upgraded.
 
-=== Install and Start ProActive using Docker Containers
+=== Automated ProActive Installation with Docker Compose
 ProActive could be easily deployed and started within a fully containerized environment. For this purpose, we provide a ready-to-use Docker Compose configuration that enables you running ProActive Workflows and Scheduling as a containerized service.
```

Comment:
we provide a ready-to-use Docker Compose configuration -- > could you 
please precise where this configuration is saved (i.e. PATH or http url) and how to access to it?

Reply:
@aminelouati it is in the next section "[2.4.1. Download the Docker Compose Installation Package](https://github.com/ow2-proactive/documentation/pull/907#_download_the_docker_compose_installation_package)"

Reply:
this is a step by step installation so the reader need to follow the installation step by step